Description: Disable networked tests.
 buildds don't have network access thus these would fail.
Author: Laszlo Boszormenyi (GCS) <gcs@debian.org>
Forwarded: no
Last-Update: 2024-03-06

---

--- a/src/platform/CMakeLists.txt
+++ b/src/platform/CMakeLists.txt
@@ -14,5 +14,4 @@ add_subdirectory(posix)
 add_subdirectory(windows)
 
 nng_test(platform_test)
-nng_test(resolver_test)
 nng_test(udp_test)
+#nng_test(resolver_test)
--- a/src/sp/transport/tcp/CMakeLists.txt
+++ b/src/sp/transport/tcp/CMakeLists.txt
@@ -14,4 +14,4 @@ nng_directory(tcp)
 nng_sources_if(NNG_TRANSPORT_TCP tcp.c)
 nng_headers_if(NNG_TRANSPORT_TCP nng/transport/tcp/tcp.h)
 nng_defines_if(NNG_TRANSPORT_TCP NNG_TRANSPORT_TCP)
-nng_test(tcp_test)
\ No newline at end of file
+#nng_test(tcp_test)
--- a/src/sp/transport/ws/CMakeLists.txt
+++ b/src/sp/transport/ws/CMakeLists.txt
@@ -19,6 +19,6 @@ nng_defines_if(NNG_TRANSPORT_WS NNG_TRAN
 nng_defines_if(NNG_TRANSPORT_WSS NNG_TRANSPORT_WSS)
 nng_sources_if(WS_ON websocket.c)
 nng_headers_if(WS_ON nng/transport/ws/websocket.h)
-nng_test_if(WS_ON ws_test)
+#nng_test_if(WS_ON ws_test)
 
 
--- a/src/supplemental/websocket/CMakeLists.txt
+++ b/src/supplemental/websocket/CMakeLists.txt
@@ -13,5 +13,5 @@ if (NNG_SUPP_WEBSOCKET)
 else ()
     nng_sources(stub.c)
 endif ()
-nng_test(wssfile_test)
-nng_test(websocket_test)
+#nng_test(wssfile_test)
+#nng_test(websocket_test)
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -126,7 +126,7 @@ endif ()
 
 add_nng_test(device 5)
 add_nng_test(files 5)
-add_nng_test1(httpclient 60 NNG_SUPP_HTTP)
+#add_nng_test1(httpclient 60 NNG_SUPP_HTTP)
 add_nng_test1(httpserver 30 NNG_SUPP_HTTP)
 add_nng_test(inproc 5)
 add_nng_test(ipc 5)
